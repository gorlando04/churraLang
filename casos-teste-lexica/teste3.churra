amigo "Jo√£o" contribui 100
amigo "Laura" contribui 80 vegetariano
amigo "Carlos" contribui 90

item carne "costela" preco_kg 40 quantidade_kg 2
item bebida preco_L 10 quantidade_L 3
item comida preco_unit 30 quantidade 1
item entretenimento preco_unit 40 quantidade 1

rateio diferente