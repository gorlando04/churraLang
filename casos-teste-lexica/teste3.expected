❌ Erro: [LÉXICO] ERRO LEXICO 'diferente' na linha 10
